cmd/helm/installer/install.go

```diff
@@ -407,3 +423,32 @@ func generateSecret(opts *Options) (*v1.Secret, error) {
 }
 
 func read(path string) (b []byte, err error) { return ioutil.ReadFile(path) }
+
+// updateEnv updates the existing env vars and allows env vars to be removed
+// by name.
+func updateEnv(existing []v1.EnvVar, env []v1.EnvVar, remove []string) []v1.EnvVar {
```

Comment:
Also, this came from the previous PR and I have a doubt now. Do we need to be able to remove env vars?

As doesn't `TILLER_HISTORY_MAX` always need to be set.